pkg/apiserver/apis/packages/types.go

```diff
@@ -38,8 +38,17 @@ type PackageSpec struct {
 	PublicName string `json:"publicName,omitempty"`
 	Version    string `json:"version,omitempty"`
 
-	DisplayName string `json:"displayName,omitempty"`
-	Description string `json:"description,omitempty"`
```

Comment:
ReleaseDate?

Reply:
it would be a timestamp though. k8s does have a pattern for naming timestamps (creationTimestamp, deletionTimestamp, ...). alternatively "at" suffix is also commonly used for representing timestamps.